src/index.js

```diff
@@ -1,10 +1,35 @@
 'use strict';
 /**
  * @module braintree-web-drop-in
+ * @description There are two ways to integrate Drop-in into your page. You can use [`dropin.create`](#.create) or a standalone script tag integration (example below).
+ *
+ * If you only need to process credit cards on your checkout form, the script tag integration is the simplest and easiest way to integrate. All you need to do is add the Dropin script inside your form where you want Drop-in to appear and include a `data-braintree-dropin-authorization` property with your [tokenization key](https://developers.braintreepayments.com/guides/authorization/tokenization-key/javascript/v3) or [client token](https://developers.braintreepayments.com/guides/authorization/client-token).
```

Comment:
Maybe we can clarify what "on your checkout form" means here. "As part of your checkout `form` element"? That doesn't sound quite right either but I think we need to emphasize that it needs to be in a form.

Also don't know if we want to say "Dropin script" or "Drop-in script" here.